regress/github-#0260/uas.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!DOCTYPE scenario SYSTEM "sipp.dtd">
+<scenario>
+  <recv request="INVITE"/>
+
+  <send retrans="500">
+    <![CDATA[
+
+      SIP/2.0 200 OK
+      Via: [last_Via:value]
+      From: [last_From:value]
+      To: [last_To:value];tag=[pid]SIPpTag00[call_number]
+      Call-ID: [last_Call-ID:value]
+      CSeq: [last_CSeq:value]
+      Contact: sip:sipp@[local_ip]:[local_port]
+      Content-Length: 0
+
+    ]]>
+  </send>
+
+  <recv request="ACK" rrs="true">
+    <action>
+      <log message="LAST Contact: [last_Contact]" />
+      <log message="LAST Via: [last_Via:value]"/>
+      <log message="LAST From: [last_From:value]"/>
+      <log message="LAST To: [last_To:value]"/>
+      <log message="LAST To:uri: [last_To:uri]"/>
+      <log message="LAST Contact:uri: [last_Contact:uri]"/>
+      <log message="LAST From-tag: [last_From:param.tag]"/>
```

Comment:
Your "key names" don't correspond to the lookups:

Suggest:
`LAST Contact|[last_Contact]`
`LAST Via:value|[last_Via:value]`
etc..